### PR TITLE
Create Day8.cpp

### PR DESCRIPTION
**Pull request by sohampathak1234**
Day 8 of our IODump Chaitra challenge!

Program to print multiplication table of a number using for loop

### CODE REVIEW

File: CPP/Day7.cpp [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* 
+
+Day 7 : Program to print multiplication table of a number using for loop
```
Comment (ganimtron-10) [inline CPP/Day7.cpp]: `Day 8` instead of `7`

File: CPP/Day7.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (ganimtron-10) [inline CPP/Day7.cpp]: File name should be `Day8.cpp`

**Review by ganimtron-10 — CHANGES_REQUESTED** [review]
Hey @sohampathak1234,
This PR is close to be merged, just look at below comments!

**Comment by sohampathak1234** [comment]
Done

**Comment by ganimtron-10** [comment]
Thanks for the changes! Merging!

**Comment by sohampathak1234** [comment]
Your Welcome!

**Pull request merged**